### ISSUE TITLE
The `PascalCamelCase` rule suddenly floods output with false reports

### ISSUE THREAD

**Issue by jhradilek** [redhat-documentation/vale-at-red-hat#427]
Following latest changes to the `RedHat` style, `vale` started flooding output with suggestions from the `PascalCamelCase` rule to wrap camel case terms in backticks. This is problematic because among few legitimate cases, a lot of these suggestions are incorrectly triggered for common Linux acronyms `DNF`, `RPM`, `KVM`, `VPN`, `IP`, `SSH` or `IBM`, and common project names like `GNOME`, `IdM`, `MariaDB`, `PostgreSQL`, `NetworkManager`, or `SELinux` that do not belong in backticks.

For reference, in documentation for RHEL 9 only 30% of reported terms use camel case (that is, contain both upper case and lower case letters).

**Comment by rolfedh** [comment]
[1090922.txt](https://github.com/redhat-documentation/vale-at-red-hat/files/10678254/1090922.txt)

**Comment by aireilly** [comment]
I've fixed the errors, sorry for the trouble. I think we should leave this issue open for now though in case there are other unintended consequences. It may also be that the rule is more trouble than it's worth. I an open to removing it :)

Also I think we should evaluate creating a "nightly" or dev release and a stable release for the Red Hat rule set. I created https://github.com/redhat-documentation/vale-at-red-hat/issues/430
